spring-boot-project/spring-boot-cli/samples/http.groovy

```diff
@@ -17,7 +16,7 @@ class Example implements CommandLineRunner {
 
 	void run(String... args) {
 		def port = context.webServer.port
-		def world = new RESTClient("http://localhost:" + port).get(path:"/").data.text
+		def world = new RestTemplate().getForObject("http://localhost:" + port + "/", String.class);
```

Comment:
```suggestion
		def world = new RestTemplate().getForObject("http://localhost:$port/", String.class);
```

It's more the Groovy way :).

Reply:
Anyway, I would put here `int` and `String` instead of just `def` to clarity.

Reply:
Thanks for taking a look, @szpak. Those are good changes to make, but they're unrelated to an upgrade to Groovy 3.0 so let's keep them separate from this PR please.